src/genegraph/transform/dosage_jira.clj

```diff
@@ -214,11 +214,12 @@
    [iri :geno/has-location (subject-iri curation)]])
 
 (defn- proposition-predicate [curation dosage]
-  (if (and (= 1 dosage)
-           (= "40: Dosage sensitivity unlikely" 
-              (get-in curation [:fields :customfield-10165 :value])))
-    :geno/BenignForCondition
-    :geno/PathogenicForCondition))
+  (let [dosage-assertion-str (if (= 1 dosage)
+                               (get-in curation [:fields :customfield-10165 :value])
+                               (get-in curation [:fields :customfield-10166 :value]))]
+    (if (= "40: Dosage sensitivity unlikely" dosage-assertion-str)
+      :geno/BenignForCondition
+      :geno/PathogenicForCondition)))
```

Comment:
This code doesn't look right to me Tristan. First of I would have expected the code to fix for the "40: Dosage sensitivity unlikely" to be up in the definition of the eveidence-levels var. Here it looks like anything else is going to be :geno/PathogenicForCondition. Right?

Reply:
This is one of the cases where we may have overcomplicated the model. Essentially 'dosage sensitivity unlikely' is sufficient evidence that the given dosage is benign for a condition. It may be worth revisiting the model, but this is the fix for now.